### PR TITLE
Add Serdes.DefaultOptions/Settings

### PR DESCRIPTION
**Pull request by bartelink**
Provide relatively easily-accessible API and guidance re integration with ASP.NET Core

### CODE REVIEW

File: src/FsCodec.SystemTextJson/Serdes.fs [hunk 1]
```diff
@@ -9,14 +9,17 @@ type Serdes private () =
...
 
+    /// Yields the settings used by <c>Serdes</c> when no <c>options</c> are supplied.
+    static member DefaultOptions : JsonSerializerOptions = defaultOptions.Value
+
     /// Serializes given value to a JSON string.
     static member Serialize<'T>
         (   /// Value to serialize.
             value : 'T,
             /// Use indentation when serializing JSON. Defaults to false.
             [<Optional; DefaultParameterValue null>] ?indent : bool) : string =
```
Comment (deviousasti) [inline src/FsCodec.SystemTextJson/Serdes.fs:20]: `false` instead of `null`? (same effect though)

**Comment by deviousasti** [comment]
Do we need both static let and lazy?

File: src/FsCodec.SystemTextJson/Serdes.fs (hunk 1 above)
Comment (bartelink) [reply src/FsCodec.SystemTextJson/Serdes.fs:20]: Doing

**Comment by bartelink** [comment]
> Do we need both static let and lazy?

I'd tend to leave it as it is - that's how Eirik wrote it long ago. I'd be reluctant to but its conceivable that one might expose a setter on the property etc which would trigger a change to the behavior

**Pull request merged**